prov/rxm/src/rxm_ep.c

```diff
@@ -744,9 +745,19 @@ rxm_ep_post_recv(struct rxm_ep *rxm_ep, const struct iovec *iov,
 	if (OFI_UNLIKELY(ret))
 		return ret;
 
-	FI_DBG(&rxm_prov, FI_LOG_EP_DATA, "Posting recv with length: %zu "
-	       "tag: 0x%" PRIx64 " ignore: 0x%" PRIx64 "\n",
-	       recv_entry->total_len, recv_entry->tag, recv_entry->ignore);
+#if ENABLE_DEBUG
+	if (recv_queue->type == RXM_RECV_QUEUE_MSG)
+		FI_DBG(&rxm_prov, FI_LOG_EP_DATA, "Posting recv with length: %zu "
+		       "addr: 0x%" PRIx64 "\n", recv_entry->total_len,
+		       recv_entry->addr);
+	else
+		FI_DBG(&rxm_prov, FI_LOG_EP_DATA, "Posting trecv with "
+		       "length: %zu addr: 0x%" PRIx64 " tag: 0x%" PRIx64
+		       " ignore: 0x%" PRIx64 "\n", recv_entry->total_len,
+		       recv_entry->addr, recv_entry->tag, recv_entry->ignore);
+#endif
```

Comment:
You may be able to drop the ENABLE_DEBUG.  The if statements will be empty and should be compiled out anyway.

Reply:
Good point. Will update.